packages/mui-base/src/Badge/Badge.tsx

```diff
@@ -94,10 +94,10 @@ const Badge = React.forwardRef(function Badge<RootComponentType extends React.El
 }) as PolymorphicComponent<BadgeTypeMap>;
 
 Badge.propTypes /* remove-proptypes */ = {
-  // ----------------------------- Warning --------------------------------
```

Comment:
These changes look unrelated, should we revert in order to have a smaller pull request?

Reply:
I had to change the `yarn` to `pnpm` in the message, so I figured I might as well fix the grammar and formatting bit. Revering those additional changes won't make the number of changed files smaller.

Reply:
Aaah, I missed that, I thought it is just the formatting. Got ya, let's keep them.